modules/apps/27-interchain-accounts/types/account_test.go

```diff
@@ -172,20 +178,14 @@ func (suite *TypesTestSuite) TestGenesisAccountValidate() {
 
 func (suite *TypesTestSuite) TestInterchainAccountMarshalYAML() {
 	addr := suite.chainA.SenderAccount.GetAddress()
-	ba := authtypes.NewBaseAccountWithAddress(addr)
+	baseAcc := authtypes.NewBaseAccountWithAddress(addr)
 
-	interchainAcc := types.NewInterchainAccount(ba, suite.chainB.SenderAccount.GetAddress().String())
-	bz, err := yaml.Marshal(types.InterchainAccountPretty{
-		Address:       addr,
-		PubKey:        "",
-		AccountNumber: interchainAcc.AccountNumber,
-		Sequence:      interchainAcc.Sequence,
-		AccountOwner:  interchainAcc.AccountOwner,
-	})
+	interchainAcc := types.NewInterchainAccount(baseAcc, suite.chainB.SenderAccount.GetAddress().String())
+	bz, err := interchainAcc.MarshalYAML()
 	suite.Require().NoError(err)
 
-	bz1, err := interchainAcc.MarshalYAML()
-	suite.Require().Equal(string(bz), string(bz1))
+	expected := fmt.Sprintf("address: %s\npublic_key: \"\"\naccount_number: 0\nsequence: 0\naccount_owner: %s\n", suite.chainA.SenderAccount.GetAddress(), suite.chainB.SenderAccount.GetAddress())
```

Comment:
Making `interchainAccountPretty` private resulted in having to re-write this test.
This is adapted from: https://github.com/cosmos/cosmos-sdk/blob/master/x/auth/types/account_test.go#L123

Reply:
yeah that's why I made it public :D Is it not better to leave it the other way?

Reply:
I think it makes more sense to maintain a "prettifying" struct as private, an external party should be unaware of its existence. 

Probably best to opt for consistency with the sdk.

Reply:
consistency with sdk is my preference